shim-server/src/main/java/org/openmhealth/shimmer/common/domain/AccessTokenDto.java

```diff
@@ -0,0 +1,28 @@
+package org.openmhealth.shimmer.common.domain;
+
+
+import javax.validation.constraints.NotNull;
+
+public class AccessTokenDto {
+
+    @NotNull
+    private long userId;
```

Comment:
This validation is of no use due to usage of primitive `long`.
Move to `Long` instead.